lib/launchers/amz.rb

```diff
@@ -616,10 +616,14 @@ def list_resource_record_sets(zone_id: nil)
     def delete_resource_records_re(re, zone_id: nil)
       logger.warn("Removing Route53 records matching #{re.inspect}")
       records = list_resource_record_sets(zone_id: zone_id)
-      to_delete = records.select { |r| r[:name] =~ re }
-      logger.debug("Removing records: #{to_delete.map {|r| r[:name]}}")
-      list = to_delete.map { |r| { action: "DELETE", resource_record_set: r } }
-      change_resource_record_sets(zone_id: zone_id, changes: list)
+      if records.count > 0
```

Comment:
I think `records` always >0, because 3.x install are sharing the same route53 zone as 4.x install.

Reply:
@jianlinliu good catch,  it should be if one line below line 620 and should be `if to_delete.count > 0`, I'll update it.